themes-default/slim/static/js/store.js

```diff
@@ -247,9 +320,21 @@ const store = new Puex({
     // Please add new mutations in the same order as the mutatinType list
     mutations: {
         [LOGIN_PENDING]() {},
-        [LOGIN_SUCCESS]() {},
-        [LOGIN_FAILED]() {},
-        [LOGOUT]() {},
+        [LOGIN_SUCCESS](state, user) {
+            state.auth.user = user;
+            state.auth.isAuthenticated = true;
+            state.auth.err = null;
+        },
+        [LOGIN_FAILED](state, err) {
```

Comment:
`err` is an object containing `{ err, credentials }` as can be seen on line 424:
https://github.com/pymedusa/Medusa/blob/b8207ce7ec20efa971bbb998933b00ca61c5e61b/themes-default/slim/static/js/store.js#L424